src/styles/components/_stars.scss

```diff
@@ -0,0 +1,19 @@
+.stars {
+  @extend %flex-between-center;
+  &__star {
+    width: 1.6rem;
+    height: 1.6rem;
+    margin-right: 4px;
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), there should be a space before the opening brace `{`.